### DISCUSSION TITLE
No data from the dashboard

### DISCUSSION THREAD

**Discussion by tdesaules** [ClusterLabs/ha_cluster_exporter#186] [Q&A] [answered]
Hi there,

I try to enable the detailed dashboard. I can't get any data from it.
I think there I don't uderstand the "job" in the doc : 

`They also assume that the target nodes in each cluster are grouped via the job label.`

I don't know how to group node based on a job label. In fact I can't find any doc regarding some label/job stuff

My corosync.conf look like : 

```
# This file is managed by Puppet.
# Changes made manually may be lost.

totem {
  version:                             2
  cluster_name:                        haproxy_10.144.118.57
  crypto_hash:                         sha1
  crypto_cipher:                       aes256
  interface {
    ringnumber:  0
    bindnetaddr: 10.144.118.57
    mcastport:   5405
  }
}

logging {
  fileline:        off
  to_stderr:       yes
  to_logfile:      yes
  to_syslog:       yes
  syslog_facility: daemon
  syslog_priority: info
  debug:           off
}

quorum {
  provider: corosync_votequorum
  two_node: 1
}

nodelist {
  node {
    ring0_addr: 10.144.118.55
    nodeid: 91
  }
  node {
    ring0_addr: 10.144.118.56
    nodeid: 92
  }
}
```

**Comment by stefanotorresi** [comment, marked as answer]
Hey, @tdesaules, what the documentation refers to is the Prometheus configuration, not the Corosync one.

More details here: https://github.com/ClusterLabs/ha_cluster_exporter/discussions/184#discussioncomment-352759

**Comment by tdesaules** [comment]
My bad, this part is good.
I have probably miss configured something because the $dc_instance variable stay null.
